core-services/prow/02_config/_config.yaml

```diff
@@ -1821,6 +1821,7 @@ tide:
     - openshift/kubernetes
     - openshift/okd-machine-os
     - openshift/origin
+    - redhat-developer/gitops-operator
```

Comment:
Move this line to be next to other `redhat-developer` repos

Reply:
@stevekuznetsov thanks. Review feedback implemented via 697be347f3b1d96b0e0d1abdb2f1b28d147c2cc1.